introduction.md

```diff
@@ -0,0 +1,3 @@
+Here I am increasing my working knowledge of git & GitHub by taking courses on CodeSchool.  I really enjoyed the Git Real & Git Real 2 courses, so I decided to take Mastering GitHub as well.
```

Comment:
I've added "Git" as my favorite Code School path even though it's not in the list you mentioned.